variants/rak4631/variant.h

```diff
@@ -228,9 +228,18 @@ SO GPIO 39/TXEN MAY NOT BE DEFINED FOR SUCCESSFUL OPERATION OF THE SX1262 - TG
 // #define PIN_GPS_EN PIN_3V3_EN
 #define PIN_GPS_PPS (17) // Pulse per second input from the GPS
 
+// On RAK2560 the GPS is be on a different UART
+#ifdef HAS_RAKPROT
+// #define GPS_RX_PIN PIN_SERIAL2_RX
+// #define GPS_TX_PIN PIN_SERIAL2_TX
+// #define PIN_GPS_EN PIN_3V3_EN
+// Disable GPS
+#define MESHTASTIC_EXCLUDE_GPS 1
+#else
```

Comment:
This does not only disable GPS but removes all GPS code from the binary. I assume goal is to have a GPS active on Serial2?

Reply:
@caveman99 Just thinking out loud, should this go into its own variant/rak2560/ - otherwise some code gets clobbered (eg, the JLink CDC pin assignment, unsure if that's used but noticed it while going through the changes)

Reply:
@Talie5in 
I can create extra variant files for RAK2560, but I wanted to avoid double maintenance, RAK2560 is just a RAK4631 on a different PCB with a solar power battery. With both in one variant.h I have to work only with one file if there are changes required.

Open for suggestions @caveman99

Reply:
really depends on how much it diverges from a stock 4631+Baseboard. Right now there's only a handful of defines. In the long term i think i'd prefer a separate variant.h file, since this is more or less a fixed configuration while we have to account for all sorts of modules on the wisblock boards. We broke out an EINK version too, since the defines were diverging too much and it got harder and harder to keep them apart.